libs/langchain-google-cloud-sql-pg/src/vectorStore.ts

```diff
@@ -179,6 +185,39 @@ class PostgresVectorStore extends VectorStore {
     )
   }
 
+  static async fromTexts(_texts: string[], _metadatas: object[], _embeddings: Embeddings, _dbConfig: dbConfigArgs): Promise<VectorStore> {
```

Comment:
We also need metadata to support object[] | object. When constructing the Document we can jus add `const metadata = Array.isArray(metadatas) ? metadatas[i] : metadatas;`